### DISCUSSION TITLE
End of phase rewards to all player in the same team/island

### DISCUSSION THREAD

**Discussion by iSeitan** [BG-Software-LLC/SuperiorSkyblock2#1543] [Approved Feature Requests] [answered]
Players that play in the same island with /is invite dont receive the end of phases rewards (like the command give money etc)

Make it so it sends to all player in the same team

**Comment by Lemvs** [comment, marked as answer]
Use /is admin cmdall, then your command with {player-name} placeholder that will be replcaed with island members names.

@OmerBenGera you can mark this answered and close it.
